lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -44,9 +45,13 @@ class FeatureCard extends React.Component {
     this.campaignsApiPage = 1;
 
     this.state = {
+      actionDisabled: false,
       campaignIndex: -1,
-      isSignedUp: ''
+      isSignedUp: false,
+      isLoading: false,
     };
+
+    this.signup = this.signup.bind(this);
```

Comment:
This [binds](function.prototype.bind.apply(...) is not a constructor) that event handler function's `this` so that it points to the component instance, rather than the DOM element. If you replaced this line with `this.signup = this`, that would overwrite the method on the object to just be a circular reference to the object itself.